app/bundles/WebhookBundle/Tests/Functional/WebhookFunctionalTest.php

```diff
@@ -43,6 +44,9 @@ protected function tearDown(): void
     public function testWebhookWorkflowWithCommandProcess()
     {
         $httpClient                    = new class() extends Client {
+            /**
+             * @var int
+             */
             public $sendRequestCounter = 0;
```

Comment:
Would this work?
```suggestion
            public int $sendRequestCounter = 0;
```

Reply:
yup, that's allowed since php 7.4. 

I'll commit it separately, as php-cs will complain due to the `=` alignment in the line above

Reply:
see https://github.com/mautic/mautic/pull/10759/commits/17895ebf1ce390e4ab72a936dff1618b5cd31563